platform/events body is undefined

## Description
Seeing an error when running `platform/events` tests:

```
1) events should handle receiving x number of events for an element instance:
     AssertionError: expected undefined not to be empty
      at doAsserterAsyncAndAddThen (node_modules/chai-as-promised/lib/chai-as-promised.js:292:33)
      at null.<anonymous> (node_modules/chai-as-promised/lib/chai-as-promised.js:282:25)
      at Object.defineProperty.get (node_modules/chakram/node_modules/chai/lib/chai/utils/overwriteProperty.js:49:37)
      at doAsserterAsyncAndAddThen (node_modules/chai-as-promised/lib/chai-as-promised.js:292:33)
      at null.<anonymous> (node_modules/chai-as-promised/lib/chai-as-promised.js:282:25)
      at Object.defineProperty.get (node_modules/chakram/node_modules/chai/lib/chai/utils/overwriteProperty.js:49:37)
      at src/test/platform/events/events.js:49:37
      at Array.forEach (native)
      at src/test/platform/events/events.js:46:20
```

## Expected Behavior
The `event.body` should be defined and contain the request body

## Steps to Reproduce
Run `churros test platform/events`

--- a/src/core/cloud.js
+++ b/src/core/cloud.js
@@ -183,17 +183,21 @@ const listenForEvents = (port, numEventsSent, waitSecs) => {
   return new Promise((resolve, reject) => {
     server = createServer((request, response) => {
         let fullBody = '';
-        request.on('data', (chunk) => fullBody += chunk.toString());
-        request.on('end', () => request.body = fullBody);
-        response.end('{}');
+        request.on('data', (chunk) => {
+          fullBody += chunk.toString()
+        });
+        request.on('end', () => {
+          request.body = fullBody
+          response.end('{}');
 
-        receivedEvents++;
-        events.push(request);
-        logger.debug('%s event(s) received', receivedEvents);
-        if (receivedEvents === numEventsSent) {
-          resolve(events);
-          server.destroy();
-        }
+          receivedEvents++;
+          events.push(request);
+          logger.debug('%s event(s) received', receivedEvents);
+          if (receivedEvents === numEventsSent) {
+            resolve(events);
+            server.destroy();
+          }
+        });
       })
       .listen(port, "localhost", (err) => {
         err ? reject(err) : logger.debug('Waiting %s seconds to receive %s events on port %s', waitSecs, numEventsSent, port);

--- a/src/core/cloud.js
+++ b/src/core/cloud.js
@@ -183,9 +183,7 @@ const listenForEvents = (port, numEventsSent, waitSecs) => {
   return new Promise((resolve, reject) => {
     server = createServer((request, response) => {
         let fullBody = '';
-        request.on('data', (chunk) => {
-          fullBody += chunk.toString()
-        });
+        request.on('data', (chunk) => fullBody += chunk.toString());
         request.on('end', () => {
           request.body = fullBody
           response.end('{}');

